userguide/install.rst

```diff
@@ -1,16 +1,17 @@
 Install Ubuntu Touch
 ====================
 
-Installing Ubuntu Touch is easy, and a lot of work has gone in to making the installation process less intimidating to less technical users. The UBports Installer is a nice graphical tool that you can use to install Ubuntu Touch on a `supported device <devices.ubports.com>`_ from your Linux, Mac or Windows computer.
+Installing Ubuntu Touch is easy, and a lot of work has gone in to making the installation process less intimidating to less technical users. The UBports Installer is a nice graphical tool that you can use to install Ubuntu Touch on a `supported device <https://ubuntu-touch.io/devices>`_ from your Linux, Mac or Windows computer.
 
 .. warning::
     If your're switching your device over from Android, you will not be able to keep any data that is currently on the device. Create an external backup if you want to keep it.
```

Comment:
While we're here, can you fix this "your're"?

Reply:
will do